web-server/web-server.markdown

```diff
@@ -170,7 +172,7 @@ Python comes with such a library called `urllib2`
 but it exposes a lot of plumbing that most people never want to care about.
 Instead,
 we recommend using the [Requests](https://pypi.python.org/pypi/requests) library.
```

Comment:
Who is "we" here? Since there's only one author suggest using "I" to avoid ambiguity, or rewriting to use passive voice.